apps/ledger-live-mobile/src/screens/Platform/App.js

```diff
@@ -31,7 +39,15 @@ const PlatformApp = ({ route }: StackScreenProps) => {
         }}
       />
     </>
-  ) : null;
+  ) : (
+    <Flex flex={1} p={10} justifyContent="center" alignItems="center">
+      {remoteLiveAppState.isLoading ? (
```

Comment:
remoteLiveApp provider will update manifests from time to time and this will interrupt the user's navigation. I think we need to display the loader when there both
- there is no value
- provider is loading